[35m[Appium][39m Welcome to Appium v1.22.3
[35m[Appium][39m Non-default server args:
[35m[Appium][39m   port: 57434
[35m[Appium][39m   loglevel: info
[35m[Appium][39m   tmpDir: C:\Users\AHADIA~1\AppData\Local\Temp\\Katalon\Appium\Temp1689060968860
[35m[Appium][39m   chromedriverPort: 57435
[35m[Appium][39m Deprecated server args:
[35m[Appium][39m   [31m--chromedriver-port[39m => --default-capabilities
[35m[Appium][39m Default capabilities, which will be added to each request unless overridden by desired capabilities:
[35m[Appium][39m   chromedriverPort: 57435
[35m[Appium][39m Appium REST http interface listener started on 0.0.0.0:57434
[35m[HTTP][39m [37m-->[39m [37mGET[39m [37m/wd/hub/status[39m
[35m[HTTP][39m [90m{}[39m
[35m[HTTP][39m [37m<-- GET /wd/hub/status [39m[32m200[39m [90m10 ms - 68[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m
[35m[HTTP][39m [90m{"desiredCapabilities":{"appPackage":"com.kfc.mobile","noReset":true,"deviceId":"","deviceName":"OPPO CPH1803 (Android 8.1.0)","platform":"ANDROID","appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","newCommandTimeout":1800,"platformVersion":"8.1.0","automationName":"uiautomator2","autoGrantPermissions":true,"platformName":"Android","udid":"cefdc881","systemPort":8281},"capabilities":{"firstMatch":[{"appium:appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","appium:appPackage":"com.kfc.mobile","appium:autoGrantPermissions":true,"appium:automationName":"uiautomator2","deviceId":"","appium:deviceName":"OPPO CPH1803 (Android 8.1.0)","appium:newCommandTimeout":1800,"appium:noReset":true,"platform":"ANDROID","platformName":"android","appium:platformVersion":"8.1.0","appium:systemPort":8281,"appium:udid":"cefdc881"}]}}[39m
[35m[BaseDriver][39m The following capabilities are not standard capabilities and should have an extension prefix:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m   chromedriverPort
[35m[Appium][39m Appium v1.22.3 creating new AndroidUiautomator2Driver (v1.70.1) session
[35m[BaseDriver][39m The following capabilities were provided, but are not recognized by Appium:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m Session created with session id: 66d19c91-6a43-474e-9059-7cecb2686393
[35m[UiAutomator2][39m Starting 'com.kfc.mobile' directly on the device
[35m[ADB][39m Found 1 'build-tools' folders under 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk' (newest first):
[35m[ADB][39m     C:/Users/Ahadian Rifky/.katalon/tools/android_sdk/build-tools/29.0.3
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m Retrieving device list
[35m[AndroidDriver][39m Using device: cefdc881
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m No app sent in, not parsing package/activity
[35m[AndroidDriver][39m Unable to set mock location for app 'io.appium.settings': Error executing adbExec. Original error: 'Command ''C:\\Users\\Ahadian Rifky\\.katalon\\tools\\android_sdk\\platform-tools\\adb.exe' -P 5037 -s cefdc881 shell appops set io.appium.settings android\:mock_location allow' exited with code 255'; Command output: Security exception: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 
[35m[AndroidDriver][39m java.lang.SecurityException: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforce(ContextImpl.java:1831)
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforcePermission(ContextImpl.java:1844)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.setMode(AppOpsService.java:674)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:2026)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService$Shell.onCommand(AppOpsService.java:1800)
[35m[AndroidDriver][39m 	at android.os.ShellCommand.exec(ShellCommand.java:96)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:1974)
[35m[AndroidDriver][39m 	at android.os.Binder.shellCommand(Binder.java:594)
[35m[AndroidDriver][39m 	at android.os.Binder.onTransact(Binder.java:492)
[35m[AndroidDriver][39m 	at com.android.internal.app.IAppOpsService$Stub.onTransact(IAppOpsService.java:341)
[35m[AndroidDriver][39m 	at android.os.Binder.execTransact(Binder.java:697)
[35m[AndroidDriver][39m 
[35m[ADB][39m Using 'apksigner.jar' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\build-tools\29.0.3\lib\apksigner.jar'
[35m[ADB][39m 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.27.0.apk' is signed with the default certificate
[35m[ADB][39m 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk' is signed with the default certificate
[35m[UiAutomator2][39m Server packages are not going to be (re)installed
[35m[ADB][39m Adding packages ["io.appium.settings","io.appium.uiautomator2.server","io.appium.uiautomator2.server.test"] to Doze whitelist
[35m[UiAutomator2][39m Starting UIAutomator2 server 4.27.0
[35m[UiAutomator2][39m Using UIAutomator2 server from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.27.0.apk' and test from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk'
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m Determined the downstream protocol as 'W3C'
[35m[AndroidDriver][39m Screen already unlocked, doing nothing
[35m[UiAutomator2][39m Starting 'com.kfc.mobile/com.kfc.mobile.presentation.splash.SplashActivity and waiting for 'com.kfc.mobile/com.kfc.mobile.presentation.splash.SplashActivity'
[35m[UiAutomator2][39m Did not get confirmation UiAutomator2 deleteSession worked; Error was: UnknownError: An unknown server-side error occurred while processing the command. Original error: 'DELETE /' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
[35m[HTTP][39m [37m<-- POST /wd/hub/session [39m[31m500[39m [90m13517 ms - 1542[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mGET[39m [37m/wd/hub/status[39m
[35m[HTTP][39m [90m{}[39m
[35m[HTTP][39m [37m<-- GET /wd/hub/status [39m[32m200[39m [90m1 ms - 68[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m
[35m[HTTP][39m [90m{"desiredCapabilities":{"appPackage":"com.kfc.mobile","noReset":true,"deviceId":"","deviceName":"OPPO CPH1803 (Android 8.1.0)","platform":"ANDROID","appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","newCommandTimeout":1800,"platformVersion":"8.1.0","automationName":"uiautomator2","autoGrantPermissions":true,"platformName":"Android","udid":"cefdc881","systemPort":8272},"capabilities":{"firstMatch":[{"appium:appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","appium:appPackage":"com.kfc.mobile","appium:autoGrantPermissions":true,"appium:automationName":"uiautomator2","deviceId":"","appium:deviceName":"OPPO CPH1803 (Android 8.1.0)","appium:newCommandTimeout":1800,"appium:noReset":true,"platform":"ANDROID","platformName":"android","appium:platformVersion":"8.1.0","appium:systemPort":8272,"appium:udid":"cefdc881"}]}}[39m
[35m[BaseDriver][39m The following capabilities are not standard capabilities and should have an extension prefix:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m   chromedriverPort
[35m[Appium][39m Appium v1.22.3 creating new AndroidUiautomator2Driver (v1.70.1) session
[35m[BaseDriver][39m The following capabilities were provided, but are not recognized by Appium:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m Session created with session id: c54caca4-d641-430f-9402-fc4c3421a5a4
[35m[UiAutomator2][39m Starting 'com.kfc.mobile' directly on the device
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m Retrieving device list
[35m[AndroidDriver][39m Using device: cefdc881
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m No app sent in, not parsing package/activity
[35m[AndroidDriver][39m Unable to set mock location for app 'io.appium.settings': Error executing adbExec. Original error: 'Command ''C:\\Users\\Ahadian Rifky\\.katalon\\tools\\android_sdk\\platform-tools\\adb.exe' -P 5037 -s cefdc881 shell appops set io.appium.settings android\:mock_location allow' exited with code 255'; Command output: Security exception: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 
[35m[AndroidDriver][39m java.lang.SecurityException: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforce(ContextImpl.java:1831)
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforcePermission(ContextImpl.java:1844)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.setMode(AppOpsService.java:674)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:2026)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService$Shell.onCommand(AppOpsService.java:1800)
[35m[AndroidDriver][39m 	at android.os.ShellCommand.exec(ShellCommand.java:96)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:1974)
[35m[AndroidDriver][39m 	at android.os.Binder.shellCommand(Binder.java:594)
[35m[AndroidDriver][39m 	at android.os.Binder.onTransact(Binder.java:492)
[35m[AndroidDriver][39m 	at com.android.internal.app.IAppOpsService$Stub.onTransact(IAppOpsService.java:341)
[35m[AndroidDriver][39m 	at android.os.Binder.execTransact(Binder.java:697)
[35m[AndroidDriver][39m 
[35m[UiAutomator2][39m Server packages are not going to be (re)installed
[35m[ADB][39m Adding packages ["io.appium.settings","io.appium.uiautomator2.server","io.appium.uiautomator2.server.test"] to Doze whitelist
[35m[UiAutomator2][39m Starting UIAutomator2 server 4.27.0
[35m[UiAutomator2][39m Using UIAutomator2 server from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.27.0.apk' and test from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk'
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m Determined the downstream protocol as 'W3C'
[35m[AndroidDriver][39m Screen already unlocked, doing nothing
[35m[UiAutomator2][39m Starting 'com.kfc.mobile/com.kfc.mobile.presentation.splash.SplashActivity and waiting for 'com.kfc.mobile/com.kfc.mobile.presentation.splash.SplashActivity'
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[UiAutomator2][39m Did not get confirmation UiAutomator2 deleteSession worked; Error was: UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
[35m[HTTP][39m [37m<-- POST /wd/hub/session [39m[31m500[39m [90m6681 ms - 1355[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mGET[39m [37m/wd/hub/status[39m
[35m[HTTP][39m [90m{}[39m
[35m[HTTP][39m [37m<-- GET /wd/hub/status [39m[32m200[39m [90m1 ms - 68[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m
[35m[HTTP][39m [90m{"desiredCapabilities":{"appPackage":"com.kfc.mobile","noReset":true,"deviceId":"","deviceName":"OPPO CPH1803 (Android 8.1.0)","platform":"ANDROID","appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","newCommandTimeout":1800,"platformVersion":"8.1.0","automationName":"uiautomator2","autoGrantPermissions":true,"platformName":"Android","udid":"cefdc881","systemPort":8297},"capabilities":{"firstMatch":[{"appium:appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","appium:appPackage":"com.kfc.mobile","appium:autoGrantPermissions":true,"appium:automationName":"uiautomator2","deviceId":"","appium:deviceName":"OPPO CPH1803 (Android 8.1.0)","appium:newCommandTimeout":1800,"appium:noReset":true,"platform":"ANDROID","platformName":"android","appium:platformVersion":"8.1.0","appium:systemPort":8297,"appium:udid":"cefdc881"}]}}[39m
[35m[BaseDriver][39m The following capabilities are not standard capabilities and should have an extension prefix:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m   chromedriverPort
[35m[Appium][39m Appium v1.22.3 creating new AndroidUiautomator2Driver (v1.70.1) session
[35m[BaseDriver][39m The following capabilities were provided, but are not recognized by Appium:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m Session created with session id: c5cd304a-b251-4fb8-9078-24f7cc36906e
[35m[UiAutomator2][39m Starting 'com.kfc.mobile' directly on the device
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m Retrieving device list
[35m[AndroidDriver][39m Using device: cefdc881
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m No app sent in, not parsing package/activity
[35m[AndroidDriver][39m Unable to set mock location for app 'io.appium.settings': Error executing adbExec. Original error: 'Command ''C:\\Users\\Ahadian Rifky\\.katalon\\tools\\android_sdk\\platform-tools\\adb.exe' -P 5037 -s cefdc881 shell appops set io.appium.settings android\:mock_location allow' exited with code 255'; Command output: Security exception: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 
[35m[AndroidDriver][39m java.lang.SecurityException: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforce(ContextImpl.java:1831)
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforcePermission(ContextImpl.java:1844)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.setMode(AppOpsService.java:674)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:2026)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService$Shell.onCommand(AppOpsService.java:1800)
[35m[AndroidDriver][39m 	at android.os.ShellCommand.exec(ShellCommand.java:96)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:1974)
[35m[AndroidDriver][39m 	at android.os.Binder.shellCommand(Binder.java:594)
[35m[AndroidDriver][39m 	at android.os.Binder.onTransact(Binder.java:492)
[35m[AndroidDriver][39m 	at com.android.internal.app.IAppOpsService$Stub.onTransact(IAppOpsService.java:341)
[35m[AndroidDriver][39m 	at android.os.Binder.execTransact(Binder.java:697)
[35m[AndroidDriver][39m 
[35m[UiAutomator2][39m Server packages are not going to be (re)installed
[35m[ADB][39m Adding packages ["io.appium.settings","io.appium.uiautomator2.server","io.appium.uiautomator2.server.test"] to Doze whitelist
[35m[UiAutomator2][39m Starting UIAutomator2 server 4.27.0
[35m[UiAutomator2][39m Using UIAutomator2 server from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.27.0.apk' and test from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk'
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m Determined the downstream protocol as 'W3C'
[35m[WD Proxy][39m Got response with status 404: {"sessionId":"c4c488ff-8e81-4a60-bd33-ef61d59c9e15","value":{"error":"invalid session id","message":"The session identified by c4c488ff-8e81-4a60-bd33-ef61d59c9e15 is not known","stacktrace":"io.appium.uiautomator2.common.exceptions.NoSuchDriverException: The session identified by c4c488ff-8e81-4a60-bd33-ef61d59c9e15 is not known\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:54)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:266)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:260)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:3...
[35m[WD Proxy][39m Got response with status 404: {"sessionId":"c4c488ff-8e81-4a60-bd33-ef61d59c9e15","value":{"error":"invalid session id","message":"The session identified by c4c488ff-8e81-4a60-bd33-ef61d59c9e15 is not known","stacktrace":"io.appium.uiautomator2.common.exceptions.NoSuchDriverException: The session identified by c4c488ff-8e81-4a60-bd33-ef61d59c9e15 is not known\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:54)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:266)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:260)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:3...
[35m[UiAutomator2][39m Did not get confirmation UiAutomator2 deleteSession worked; Error was: NoSuchDriverError: The session identified by c4c488ff-8e81-4a60-bd33-ef61d59c9e15 is not known
[35m[HTTP][39m [37m<-- POST /wd/hub/session [39m[33m404[39m [90m3282 ms - 3691[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mGET[39m [37m/wd/hub/status[39m
[35m[HTTP][39m [90m{}[39m
[35m[HTTP][39m [37m<-- GET /wd/hub/status [39m[32m200[39m [90m1 ms - 68[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m
[35m[HTTP][39m [90m{"desiredCapabilities":{"appPackage":"com.kfc.mobile","noReset":true,"deviceId":"","deviceName":"OPPO CPH1803 (Android 8.1.0)","platform":"ANDROID","appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","newCommandTimeout":1800,"platformVersion":"8.1.0","automationName":"uiautomator2","autoGrantPermissions":true,"platformName":"Android","udid":"cefdc881","systemPort":8255},"capabilities":{"firstMatch":[{"appium:appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","appium:appPackage":"com.kfc.mobile","appium:autoGrantPermissions":true,"appium:automationName":"uiautomator2","deviceId":"","appium:deviceName":"OPPO CPH1803 (Android 8.1.0)","appium:newCommandTimeout":1800,"appium:noReset":true,"platform":"ANDROID","platformName":"android","appium:platformVersion":"8.1.0","appium:systemPort":8255,"appium:udid":"cefdc881"}]}}[39m
[35m[BaseDriver][39m The following capabilities are not standard capabilities and should have an extension prefix:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m   chromedriverPort
[35m[Appium][39m Appium v1.22.3 creating new AndroidUiautomator2Driver (v1.70.1) session
[35m[BaseDriver][39m The following capabilities were provided, but are not recognized by Appium:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m Session created with session id: 05ad6b5b-8dfb-4be0-b986-7201f5ec6836
[35m[UiAutomator2][39m Starting 'com.kfc.mobile' directly on the device
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m Retrieving device list
[35m[AndroidDriver][39m Using device: cefdc881
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m No app sent in, not parsing package/activity
[35m[AndroidDriver][39m Unable to set mock location for app 'io.appium.settings': Error executing adbExec. Original error: 'Command ''C:\\Users\\Ahadian Rifky\\.katalon\\tools\\android_sdk\\platform-tools\\adb.exe' -P 5037 -s cefdc881 shell appops set io.appium.settings android\:mock_location allow' exited with code 255'; Command output: Security exception: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 
[35m[AndroidDriver][39m java.lang.SecurityException: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforce(ContextImpl.java:1831)
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforcePermission(ContextImpl.java:1844)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.setMode(AppOpsService.java:674)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:2026)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService$Shell.onCommand(AppOpsService.java:1800)
[35m[AndroidDriver][39m 	at android.os.ShellCommand.exec(ShellCommand.java:96)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:1974)
[35m[AndroidDriver][39m 	at android.os.Binder.shellCommand(Binder.java:594)
[35m[AndroidDriver][39m 	at android.os.Binder.onTransact(Binder.java:492)
[35m[AndroidDriver][39m 	at com.android.internal.app.IAppOpsService$Stub.onTransact(IAppOpsService.java:341)
[35m[AndroidDriver][39m 	at android.os.Binder.execTransact(Binder.java:697)
[35m[AndroidDriver][39m 
[35m[UiAutomator2][39m Server packages are not going to be (re)installed
[35m[ADB][39m Adding packages ["io.appium.settings","io.appium.uiautomator2.server","io.appium.uiautomator2.server.test"] to Doze whitelist
[35m[UiAutomator2][39m Starting UIAutomator2 server 4.27.0
[35m[UiAutomator2][39m Using UIAutomator2 server from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.27.0.apk' and test from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk'
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[UiAutomator2][39m The instrumentation process has been unexpectedly terminated. Retrying UiAutomator2 startup (#1 of 1)
[35m[ADB][39m No 'uiautomator' process has been found
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m Determined the downstream protocol as 'W3C'
[35m[WD Proxy][39m Got response with status 404: {"sessionId":"2a284b86-5509-4842-8b2d-852cecff7fab","value":{"error":"invalid session id","message":"The session identified by 2a284b86-5509-4842-8b2d-852cecff7fab is not known","stacktrace":"io.appium.uiautomator2.common.exceptions.NoSuchDriverException: The session identified by 2a284b86-5509-4842-8b2d-852cecff7fab is not known\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:54)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:266)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:260)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:3...
[35m[WD Proxy][39m Got response with status 404: {"sessionId":"2a284b86-5509-4842-8b2d-852cecff7fab","value":{"error":"invalid session id","message":"The session identified by 2a284b86-5509-4842-8b2d-852cecff7fab is not known","stacktrace":"io.appium.uiautomator2.common.exceptions.NoSuchDriverException: The session identified by 2a284b86-5509-4842-8b2d-852cecff7fab is not known\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:54)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:266)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:260)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:3...
[35m[UiAutomator2][39m Did not get confirmation UiAutomator2 deleteSession worked; Error was: NoSuchDriverError: The session identified by 2a284b86-5509-4842-8b2d-852cecff7fab is not known
[35m[HTTP][39m [37m<-- POST /wd/hub/session [39m[33m404[39m [90m11838 ms - 3691[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mGET[39m [37m/wd/hub/status[39m
[35m[HTTP][39m [90m{}[39m
[35m[HTTP][39m [37m<-- GET /wd/hub/status [39m[32m200[39m [90m1 ms - 68[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m
[35m[HTTP][39m [90m{"desiredCapabilities":{"appPackage":"com.kfc.mobile","noReset":true,"deviceId":"","deviceName":"OPPO CPH1803 (Android 8.1.0)","platform":"ANDROID","appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","newCommandTimeout":1800,"platformVersion":"8.1.0","automationName":"uiautomator2","autoGrantPermissions":true,"platformName":"Android","udid":"cefdc881","systemPort":8233},"capabilities":{"firstMatch":[{"appium:appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","appium:appPackage":"com.kfc.mobile","appium:autoGrantPermissions":true,"appium:automationName":"uiautomator2","deviceId":"","appium:deviceName":"OPPO CPH1803 (Android 8.1.0)","appium:newCommandTimeout":1800,"appium:noReset":true,"platform":"ANDROID","platformName":"android","appium:platformVersion":"8.1.0","appium:systemPort":8233,"appium:udid":"cefdc881"}]}}[39m
[35m[BaseDriver][39m The following capabilities are not standard capabilities and should have an extension prefix:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m   chromedriverPort
[35m[Appium][39m Appium v1.22.3 creating new AndroidUiautomator2Driver (v1.70.1) session
[35m[BaseDriver][39m The following capabilities were provided, but are not recognized by Appium:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m Session created with session id: ece2d7ac-9a38-4f40-aeb4-6065c8947b96
[35m[UiAutomator2][39m Starting 'com.kfc.mobile' directly on the device
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m Retrieving device list
[35m[AndroidDriver][39m Using device: cefdc881
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m No app sent in, not parsing package/activity
[35m[AndroidDriver][39m Unable to set mock location for app 'io.appium.settings': Error executing adbExec. Original error: 'Command ''C:\\Users\\Ahadian Rifky\\.katalon\\tools\\android_sdk\\platform-tools\\adb.exe' -P 5037 -s cefdc881 shell appops set io.appium.settings android\:mock_location allow' exited with code 255'; Command output: Security exception: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 
[35m[AndroidDriver][39m java.lang.SecurityException: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforce(ContextImpl.java:1831)
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforcePermission(ContextImpl.java:1844)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.setMode(AppOpsService.java:674)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:2026)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService$Shell.onCommand(AppOpsService.java:1800)
[35m[AndroidDriver][39m 	at android.os.ShellCommand.exec(ShellCommand.java:96)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:1974)
[35m[AndroidDriver][39m 	at android.os.Binder.shellCommand(Binder.java:594)
[35m[AndroidDriver][39m 	at android.os.Binder.onTransact(Binder.java:492)
[35m[AndroidDriver][39m 	at com.android.internal.app.IAppOpsService$Stub.onTransact(IAppOpsService.java:341)
[35m[AndroidDriver][39m 	at android.os.Binder.execTransact(Binder.java:697)
[35m[AndroidDriver][39m 
[35m[UiAutomator2][39m Server packages are not going to be (re)installed
[35m[ADB][39m Adding packages ["io.appium.settings","io.appium.uiautomator2.server","io.appium.uiautomator2.server.test"] to Doze whitelist
[35m[UiAutomator2][39m Starting UIAutomator2 server 4.27.0
[35m[UiAutomator2][39m Using UIAutomator2 server from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.27.0.apk' and test from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk'
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[UiAutomator2][39m The instrumentation process has been unexpectedly terminated. Retrying UiAutomator2 startup (#1 of 1)
[35m[ADB][39m No 'uiautomator' process has been found
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m Determined the downstream protocol as 'W3C'
[35m[WD Proxy][39m Got response with status 404: {"sessionId":"76b68f6e-24b1-4e85-ab9f-d15fdaec41dc","value":{"error":"invalid session id","message":"The session identified by 76b68f6e-24b1-4e85-ab9f-d15fdaec41dc is not known","stacktrace":"io.appium.uiautomator2.common.exceptions.NoSuchDriverException: The session identified by 76b68f6e-24b1-4e85-ab9f-d15fdaec41dc is not known\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:54)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:266)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:260)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:3...
[35m[WD Proxy][39m Got response with status 404: {"sessionId":"76b68f6e-24b1-4e85-ab9f-d15fdaec41dc","value":{"error":"invalid session id","message":"The session identified by 76b68f6e-24b1-4e85-ab9f-d15fdaec41dc is not known","stacktrace":"io.appium.uiautomator2.common.exceptions.NoSuchDriverException: The session identified by 76b68f6e-24b1-4e85-ab9f-d15fdaec41dc is not known\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:54)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:266)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:260)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:3...
[35m[UiAutomator2][39m Did not get confirmation UiAutomator2 deleteSession worked; Error was: NoSuchDriverError: The session identified by 76b68f6e-24b1-4e85-ab9f-d15fdaec41dc is not known
[35m[HTTP][39m [37m<-- POST /wd/hub/session [39m[33m404[39m [90m11858 ms - 3691[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mGET[39m [37m/wd/hub/status[39m
[35m[HTTP][39m [90m{}[39m
[35m[HTTP][39m [37m<-- GET /wd/hub/status [39m[32m200[39m [90m1 ms - 68[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m
[35m[HTTP][39m [90m{"desiredCapabilities":{"appPackage":"com.kfc.mobile","noReset":true,"deviceId":"","deviceName":"OPPO CPH1803 (Android 8.1.0)","platform":"ANDROID","appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","newCommandTimeout":1800,"platformVersion":"8.1.0","automationName":"uiautomator2","autoGrantPermissions":true,"platformName":"Android","udid":"cefdc881","systemPort":8287},"capabilities":{"firstMatch":[{"appium:appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","appium:appPackage":"com.kfc.mobile","appium:autoGrantPermissions":true,"appium:automationName":"uiautomator2","deviceId":"","appium:deviceName":"OPPO CPH1803 (Android 8.1.0)","appium:newCommandTimeout":1800,"appium:noReset":true,"platform":"ANDROID","platformName":"android","appium:platformVersion":"8.1.0","appium:systemPort":8287,"appium:udid":"cefdc881"}]}}[39m
[35m[BaseDriver][39m The following capabilities are not standard capabilities and should have an extension prefix:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m   chromedriverPort
[35m[Appium][39m Appium v1.22.3 creating new AndroidUiautomator2Driver (v1.70.1) session
[35m[BaseDriver][39m The following capabilities were provided, but are not recognized by Appium:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m Session created with session id: 25aefde9-0a18-4121-bf94-ecf59c024b46
[35m[UiAutomator2][39m Starting 'com.kfc.mobile' directly on the device
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m Retrieving device list
[35m[AndroidDriver][39m Using device: cefdc881
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m No app sent in, not parsing package/activity
[35m[AndroidDriver][39m Unable to set mock location for app 'io.appium.settings': Error executing adbExec. Original error: 'Command ''C:\\Users\\Ahadian Rifky\\.katalon\\tools\\android_sdk\\platform-tools\\adb.exe' -P 5037 -s cefdc881 shell appops set io.appium.settings android\:mock_location allow' exited with code 255'; Command output: Security exception: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 
[35m[AndroidDriver][39m java.lang.SecurityException: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforce(ContextImpl.java:1831)
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforcePermission(ContextImpl.java:1844)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.setMode(AppOpsService.java:674)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:2026)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService$Shell.onCommand(AppOpsService.java:1800)
[35m[AndroidDriver][39m 	at android.os.ShellCommand.exec(ShellCommand.java:96)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:1974)
[35m[AndroidDriver][39m 	at android.os.Binder.shellCommand(Binder.java:594)
[35m[AndroidDriver][39m 	at android.os.Binder.onTransact(Binder.java:492)
[35m[AndroidDriver][39m 	at com.android.internal.app.IAppOpsService$Stub.onTransact(IAppOpsService.java:341)
[35m[AndroidDriver][39m 	at android.os.Binder.execTransact(Binder.java:697)
[35m[AndroidDriver][39m 
[35m[UiAutomator2][39m Server packages are not going to be (re)installed
[35m[ADB][39m Adding packages ["io.appium.settings","io.appium.uiautomator2.server","io.appium.uiautomator2.server.test"] to Doze whitelist
[35m[UiAutomator2][39m Starting UIAutomator2 server 4.27.0
[35m[UiAutomator2][39m Using UIAutomator2 server from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.27.0.apk' and test from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk'
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[UiAutomator2][39m The instrumentation process has been unexpectedly terminated. Retrying UiAutomator2 startup (#1 of 1)
[35m[ADB][39m No 'uiautomator' process has been found
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m Determined the downstream protocol as 'W3C'
[35m[WD Proxy][39m Got response with status 404: {"sessionId":"d99103c9-98a7-42bb-9d07-ae3360ad0527","value":{"error":"invalid session id","message":"The session identified by d99103c9-98a7-42bb-9d07-ae3360ad0527 is not known","stacktrace":"io.appium.uiautomator2.common.exceptions.NoSuchDriverException: The session identified by d99103c9-98a7-42bb-9d07-ae3360ad0527 is not known\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:54)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:266)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:260)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:3...
[35m[WD Proxy][39m Got response with status 404: {"sessionId":"d99103c9-98a7-42bb-9d07-ae3360ad0527","value":{"error":"invalid session id","message":"The session identified by d99103c9-98a7-42bb-9d07-ae3360ad0527 is not known","stacktrace":"io.appium.uiautomator2.common.exceptions.NoSuchDriverException: The session identified by d99103c9-98a7-42bb-9d07-ae3360ad0527 is not known\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:54)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:266)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:260)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:3...
[35m[UiAutomator2][39m Did not get confirmation UiAutomator2 deleteSession worked; Error was: NoSuchDriverError: The session identified by d99103c9-98a7-42bb-9d07-ae3360ad0527 is not known
[35m[HTTP][39m [37m<-- POST /wd/hub/session [39m[33m404[39m [90m11942 ms - 3691[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mGET[39m [37m/wd/hub/status[39m
[35m[HTTP][39m [90m{}[39m
[35m[HTTP][39m [37m<-- GET /wd/hub/status [39m[32m200[39m [90m1 ms - 68[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m
[35m[HTTP][39m [90m{"desiredCapabilities":{"appPackage":"com.kfc.mobile","noReset":true,"deviceId":"","deviceName":"OPPO CPH1803 (Android 8.1.0)","platform":"ANDROID","appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","newCommandTimeout":1800,"platformVersion":"8.1.0","automationName":"uiautomator2","autoGrantPermissions":true,"platformName":"Android","udid":"cefdc881","systemPort":8290},"capabilities":{"firstMatch":[{"appium:appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","appium:appPackage":"com.kfc.mobile","appium:autoGrantPermissions":true,"appium:automationName":"uiautomator2","deviceId":"","appium:deviceName":"OPPO CPH1803 (Android 8.1.0)","appium:newCommandTimeout":1800,"appium:noReset":true,"platform":"ANDROID","platformName":"android","appium:platformVersion":"8.1.0","appium:systemPort":8290,"appium:udid":"cefdc881"}]}}[39m
[35m[BaseDriver][39m The following capabilities are not standard capabilities and should have an extension prefix:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m   chromedriverPort
[35m[Appium][39m Appium v1.22.3 creating new AndroidUiautomator2Driver (v1.70.1) session
[35m[BaseDriver][39m The following capabilities were provided, but are not recognized by Appium:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m Session created with session id: 0310a937-fadf-43a0-bff9-3949e81188b1
[35m[UiAutomator2][39m Starting 'com.kfc.mobile' directly on the device
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m Retrieving device list
[35m[AndroidDriver][39m Using device: cefdc881
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m No app sent in, not parsing package/activity
[35m[AndroidDriver][39m Unable to set mock location for app 'io.appium.settings': Error executing adbExec. Original error: 'Command ''C:\\Users\\Ahadian Rifky\\.katalon\\tools\\android_sdk\\platform-tools\\adb.exe' -P 5037 -s cefdc881 shell appops set io.appium.settings android\:mock_location allow' exited with code 255'; Command output: Security exception: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 
[35m[AndroidDriver][39m java.lang.SecurityException: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforce(ContextImpl.java:1831)
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforcePermission(ContextImpl.java:1844)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.setMode(AppOpsService.java:674)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:2026)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService$Shell.onCommand(AppOpsService.java:1800)
[35m[AndroidDriver][39m 	at android.os.ShellCommand.exec(ShellCommand.java:96)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:1974)
[35m[AndroidDriver][39m 	at android.os.Binder.shellCommand(Binder.java:594)
[35m[AndroidDriver][39m 	at android.os.Binder.onTransact(Binder.java:492)
[35m[AndroidDriver][39m 	at com.android.internal.app.IAppOpsService$Stub.onTransact(IAppOpsService.java:341)
[35m[AndroidDriver][39m 	at android.os.Binder.execTransact(Binder.java:697)
[35m[AndroidDriver][39m 
[35m[UiAutomator2][39m Server packages are not going to be (re)installed
[35m[ADB][39m Adding packages ["io.appium.settings","io.appium.uiautomator2.server","io.appium.uiautomator2.server.test"] to Doze whitelist
[35m[UiAutomator2][39m Starting UIAutomator2 server 4.27.0
[35m[UiAutomator2][39m Using UIAutomator2 server from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.27.0.apk' and test from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk'
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[UiAutomator2][39m The instrumentation process has been unexpectedly terminated. Retrying UiAutomator2 startup (#1 of 1)
[35m[ADB][39m No 'uiautomator' process has been found
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m Determined the downstream protocol as 'W3C'
[35m[WD Proxy][39m Got response with status 404: {"sessionId":"37e56b1a-1e09-4ed5-9643-3d78b3bb5f9e","value":{"error":"invalid session id","message":"The session identified by 37e56b1a-1e09-4ed5-9643-3d78b3bb5f9e is not known","stacktrace":"io.appium.uiautomator2.common.exceptions.NoSuchDriverException: The session identified by 37e56b1a-1e09-4ed5-9643-3d78b3bb5f9e is not known\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:54)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:266)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:260)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:3...
[35m[WD Proxy][39m Got response with status 404: {"sessionId":"37e56b1a-1e09-4ed5-9643-3d78b3bb5f9e","value":{"error":"invalid session id","message":"The session identified by 37e56b1a-1e09-4ed5-9643-3d78b3bb5f9e is not known","stacktrace":"io.appium.uiautomator2.common.exceptions.NoSuchDriverException: The session identified by 37e56b1a-1e09-4ed5-9643-3d78b3bb5f9e is not known\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:54)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:266)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:260)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:3...
[35m[UiAutomator2][39m Did not get confirmation UiAutomator2 deleteSession worked; Error was: NoSuchDriverError: The session identified by 37e56b1a-1e09-4ed5-9643-3d78b3bb5f9e is not known
[35m[HTTP][39m [37m<-- POST /wd/hub/session [39m[33m404[39m [90m11925 ms - 3691[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mGET[39m [37m/wd/hub/status[39m
[35m[HTTP][39m [90m{}[39m
[35m[HTTP][39m [37m<-- GET /wd/hub/status [39m[32m200[39m [90m1 ms - 68[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m
[35m[HTTP][39m [90m{"desiredCapabilities":{"appPackage":"com.kfc.mobile","noReset":true,"deviceId":"","deviceName":"OPPO CPH1803 (Android 8.1.0)","platform":"ANDROID","appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","newCommandTimeout":1800,"platformVersion":"8.1.0","automationName":"uiautomator2","autoGrantPermissions":true,"platformName":"Android","udid":"cefdc881","systemPort":8201},"capabilities":{"firstMatch":[{"appium:appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","appium:appPackage":"com.kfc.mobile","appium:autoGrantPermissions":true,"appium:automationName":"uiautomator2","deviceId":"","appium:deviceName":"OPPO CPH1803 (Android 8.1.0)","appium:newCommandTimeout":1800,"appium:noReset":true,"platform":"ANDROID","platformName":"android","appium:platformVersion":"8.1.0","appium:systemPort":8201,"appium:udid":"cefdc881"}]}}[39m
[35m[BaseDriver][39m The following capabilities are not standard capabilities and should have an extension prefix:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m   chromedriverPort
[35m[Appium][39m Appium v1.22.3 creating new AndroidUiautomator2Driver (v1.70.1) session
[35m[BaseDriver][39m The following capabilities were provided, but are not recognized by Appium:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m Session created with session id: efcc979e-277b-42f4-bd51-a4e2984c0ad3
[35m[UiAutomator2][39m Starting 'com.kfc.mobile' directly on the device
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m Retrieving device list
[35m[AndroidDriver][39m Using device: cefdc881
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m No app sent in, not parsing package/activity
[35m[AndroidDriver][39m Unable to set mock location for app 'io.appium.settings': Error executing adbExec. Original error: 'Command ''C:\\Users\\Ahadian Rifky\\.katalon\\tools\\android_sdk\\platform-tools\\adb.exe' -P 5037 -s cefdc881 shell appops set io.appium.settings android\:mock_location allow' exited with code 255'; Command output: Security exception: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 
[35m[AndroidDriver][39m java.lang.SecurityException: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforce(ContextImpl.java:1831)
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforcePermission(ContextImpl.java:1844)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.setMode(AppOpsService.java:674)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:2026)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService$Shell.onCommand(AppOpsService.java:1800)
[35m[AndroidDriver][39m 	at android.os.ShellCommand.exec(ShellCommand.java:96)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:1974)
[35m[AndroidDriver][39m 	at android.os.Binder.shellCommand(Binder.java:594)
[35m[AndroidDriver][39m 	at android.os.Binder.onTransact(Binder.java:492)
[35m[AndroidDriver][39m 	at com.android.internal.app.IAppOpsService$Stub.onTransact(IAppOpsService.java:341)
[35m[AndroidDriver][39m 	at android.os.Binder.execTransact(Binder.java:697)
[35m[AndroidDriver][39m 
[35m[UiAutomator2][39m Server packages are not going to be (re)installed
[35m[ADB][39m Adding packages ["io.appium.settings","io.appium.uiautomator2.server","io.appium.uiautomator2.server.test"] to Doze whitelist
[35m[UiAutomator2][39m Starting UIAutomator2 server 4.27.0
[35m[UiAutomator2][39m Using UIAutomator2 server from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.27.0.apk' and test from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk'
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[UiAutomator2][39m The instrumentation process has been unexpectedly terminated. Retrying UiAutomator2 startup (#1 of 1)
[35m[ADB][39m No 'uiautomator' process has been found
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m Determined the downstream protocol as 'W3C'
[35m[AndroidDriver][39m Screen already unlocked, doing nothing
[35m[UiAutomator2][39m Starting 'com.kfc.mobile/com.kfc.mobile.presentation.splash.SplashActivity and waiting for 'com.kfc.mobile/com.kfc.mobile.presentation.splash.SplashActivity'
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[UiAutomator2][39m Did not get confirmation UiAutomator2 deleteSession worked; Error was: UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
[35m[HTTP][39m [37m<-- POST /wd/hub/session [39m[31m500[39m [90m15492 ms - 1355[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mGET[39m [37m/wd/hub/status[39m
[35m[HTTP][39m [90m{}[39m
[35m[HTTP][39m [37m<-- GET /wd/hub/status [39m[32m200[39m [90m1 ms - 68[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m
[35m[HTTP][39m [90m{"desiredCapabilities":{"appPackage":"com.kfc.mobile","noReset":true,"deviceId":"","deviceName":"OPPO CPH1803 (Android 8.1.0)","platform":"ANDROID","appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","newCommandTimeout":1800,"platformVersion":"8.1.0","automationName":"uiautomator2","autoGrantPermissions":true,"platformName":"Android","udid":"cefdc881","systemPort":8291},"capabilities":{"firstMatch":[{"appium:appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","appium:appPackage":"com.kfc.mobile","appium:autoGrantPermissions":true,"appium:automationName":"uiautomator2","deviceId":"","appium:deviceName":"OPPO CPH1803 (Android 8.1.0)","appium:newCommandTimeout":1800,"appium:noReset":true,"platform":"ANDROID","platformName":"android","appium:platformVersion":"8.1.0","appium:systemPort":8291,"appium:udid":"cefdc881"}]}}[39m
[35m[BaseDriver][39m The following capabilities are not standard capabilities and should have an extension prefix:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m   chromedriverPort
[35m[Appium][39m Appium v1.22.3 creating new AndroidUiautomator2Driver (v1.70.1) session
[35m[BaseDriver][39m The following capabilities were provided, but are not recognized by Appium:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m Session created with session id: 87fd819f-053d-4636-b252-8e13c509c8d8
[35m[UiAutomator2][39m Starting 'com.kfc.mobile' directly on the device
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m Retrieving device list
[35m[AndroidDriver][39m Using device: cefdc881
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m No app sent in, not parsing package/activity
[35m[AndroidDriver][39m Unable to set mock location for app 'io.appium.settings': Error executing adbExec. Original error: 'Command ''C:\\Users\\Ahadian Rifky\\.katalon\\tools\\android_sdk\\platform-tools\\adb.exe' -P 5037 -s cefdc881 shell appops set io.appium.settings android\:mock_location allow' exited with code 255'; Command output: Security exception: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 
[35m[AndroidDriver][39m java.lang.SecurityException: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforce(ContextImpl.java:1831)
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforcePermission(ContextImpl.java:1844)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.setMode(AppOpsService.java:674)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:2026)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService$Shell.onCommand(AppOpsService.java:1800)
[35m[AndroidDriver][39m 	at android.os.ShellCommand.exec(ShellCommand.java:96)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:1974)
[35m[AndroidDriver][39m 	at android.os.Binder.shellCommand(Binder.java:594)
[35m[AndroidDriver][39m 	at android.os.Binder.onTransact(Binder.java:492)
[35m[AndroidDriver][39m 	at com.android.internal.app.IAppOpsService$Stub.onTransact(IAppOpsService.java:341)
[35m[AndroidDriver][39m 	at android.os.Binder.execTransact(Binder.java:697)
[35m[AndroidDriver][39m 
[35m[UiAutomator2][39m Server packages are not going to be (re)installed
[35m[ADB][39m Adding packages ["io.appium.settings","io.appium.uiautomator2.server","io.appium.uiautomator2.server.test"] to Doze whitelist
[35m[UiAutomator2][39m Starting UIAutomator2 server 4.27.0
[35m[UiAutomator2][39m Using UIAutomator2 server from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.27.0.apk' and test from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk'
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m Determined the downstream protocol as 'W3C'
[35m[AndroidDriver][39m Screen already unlocked, doing nothing
[35m[UiAutomator2][39m Starting 'com.kfc.mobile/com.kfc.mobile.presentation.splash.SplashActivity and waiting for 'com.kfc.mobile/com.kfc.mobile.presentation.splash.SplashActivity'
[35m[UiAutomator2][39m Did not get confirmation UiAutomator2 deleteSession worked; Error was: UnknownError: An unknown server-side error occurred while processing the command. Original error: 'DELETE /' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
[35m[HTTP][39m [37m<-- POST /wd/hub/session [39m[31m500[39m [90m11613 ms - 1542[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mGET[39m [37m/wd/hub/status[39m
[35m[HTTP][39m [90m{}[39m
[35m[HTTP][39m [37m<-- GET /wd/hub/status [39m[32m200[39m [90m2 ms - 68[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m
[35m[HTTP][39m [90m{"desiredCapabilities":{"appPackage":"com.kfc.mobile","noReset":true,"deviceId":"","deviceName":"OPPO CPH1803 (Android 8.1.0)","platform":"ANDROID","appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","newCommandTimeout":1800,"platformVersion":"8.1.0","automationName":"uiautomator2","autoGrantPermissions":true,"platformName":"Android","udid":"cefdc881","systemPort":8271},"capabilities":{"firstMatch":[{"appium:appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","appium:appPackage":"com.kfc.mobile","appium:autoGrantPermissions":true,"appium:automationName":"uiautomator2","deviceId":"","appium:deviceName":"OPPO CPH1803 (Android 8.1.0)","appium:newCommandTimeout":1800,"appium:noReset":true,"platform":"ANDROID","platformName":"android","appium:platformVersion":"8.1.0","appium:systemPort":8271,"appium:udid":"cefdc881"}]}}[39m
[35m[BaseDriver][39m The following capabilities are not standard capabilities and should have an extension prefix:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m   chromedriverPort
[35m[Appium][39m Appium v1.22.3 creating new AndroidUiautomator2Driver (v1.70.1) session
[35m[BaseDriver][39m The following capabilities were provided, but are not recognized by Appium:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m Session created with session id: 67aa17d6-1ec6-4938-86ef-8d2070361b6e
[35m[UiAutomator2][39m Starting 'com.kfc.mobile' directly on the device
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m Retrieving device list
[35m[AndroidDriver][39m Using device: cefdc881
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m No app sent in, not parsing package/activity
[35m[AndroidDriver][39m Unable to set mock location for app 'io.appium.settings': Error executing adbExec. Original error: 'Command ''C:\\Users\\Ahadian Rifky\\.katalon\\tools\\android_sdk\\platform-tools\\adb.exe' -P 5037 -s cefdc881 shell appops set io.appium.settings android\:mock_location allow' exited with code 255'; Command output: Security exception: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 
[35m[AndroidDriver][39m java.lang.SecurityException: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforce(ContextImpl.java:1831)
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforcePermission(ContextImpl.java:1844)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.setMode(AppOpsService.java:674)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:2026)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService$Shell.onCommand(AppOpsService.java:1800)
[35m[AndroidDriver][39m 	at android.os.ShellCommand.exec(ShellCommand.java:96)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:1974)
[35m[AndroidDriver][39m 	at android.os.Binder.shellCommand(Binder.java:594)
[35m[AndroidDriver][39m 	at android.os.Binder.onTransact(Binder.java:492)
[35m[AndroidDriver][39m 	at com.android.internal.app.IAppOpsService$Stub.onTransact(IAppOpsService.java:341)
[35m[AndroidDriver][39m 	at android.os.Binder.execTransact(Binder.java:697)
[35m[AndroidDriver][39m 
[35m[UiAutomator2][39m Server packages are not going to be (re)installed
[35m[ADB][39m Adding packages ["io.appium.settings","io.appium.uiautomator2.server","io.appium.uiautomator2.server.test"] to Doze whitelist
[35m[UiAutomator2][39m Starting UIAutomator2 server 4.27.0
[35m[UiAutomator2][39m Using UIAutomator2 server from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.27.0.apk' and test from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk'
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m Determined the downstream protocol as 'W3C'
[35m[AndroidDriver][39m Screen already unlocked, doing nothing
[35m[UiAutomator2][39m Starting 'com.kfc.mobile/com.kfc.mobile.presentation.splash.SplashActivity and waiting for 'com.kfc.mobile/com.kfc.mobile.presentation.splash.SplashActivity'
[35m[UiAutomator2][39m Did not get confirmation UiAutomator2 deleteSession worked; Error was: UnknownError: An unknown server-side error occurred while processing the command. Original error: 'DELETE /' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
[35m[HTTP][39m [37m<-- POST /wd/hub/session [39m[31m500[39m [90m6733 ms - 1542[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mGET[39m [37m/wd/hub/status[39m
[35m[HTTP][39m [90m{}[39m
[35m[HTTP][39m [37m<-- GET /wd/hub/status [39m[32m200[39m [90m1 ms - 68[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m
[35m[HTTP][39m [90m{"desiredCapabilities":{"appPackage":"com.kfc.mobile","noReset":true,"deviceId":"","deviceName":"OPPO CPH1803 (Android 8.1.0)","platform":"ANDROID","appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","newCommandTimeout":1800,"platformVersion":"8.1.0","automationName":"uiautomator2","autoGrantPermissions":true,"platformName":"Android","udid":"cefdc881","systemPort":8244},"capabilities":{"firstMatch":[{"appium:appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","appium:appPackage":"com.kfc.mobile","appium:autoGrantPermissions":true,"appium:automationName":"uiautomator2","deviceId":"","appium:deviceName":"OPPO CPH1803 (Android 8.1.0)","appium:newCommandTimeout":1800,"appium:noReset":true,"platform":"ANDROID","platformName":"android","appium:platformVersion":"8.1.0","appium:systemPort":8244,"appium:udid":"cefdc881"}]}}[39m
[35m[BaseDriver][39m The following capabilities are not standard capabilities and should have an extension prefix:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m   chromedriverPort
[35m[Appium][39m Appium v1.22.3 creating new AndroidUiautomator2Driver (v1.70.1) session
[35m[BaseDriver][39m The following capabilities were provided, but are not recognized by Appium:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m Session created with session id: 348c5809-ec93-417e-aa23-5e5756138156
[35m[UiAutomator2][39m Starting 'com.kfc.mobile' directly on the device
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m Retrieving device list
[35m[AndroidDriver][39m Using device: cefdc881
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m No app sent in, not parsing package/activity
[35m[AndroidDriver][39m Unable to set mock location for app 'io.appium.settings': Error executing adbExec. Original error: 'Command ''C:\\Users\\Ahadian Rifky\\.katalon\\tools\\android_sdk\\platform-tools\\adb.exe' -P 5037 -s cefdc881 shell appops set io.appium.settings android\:mock_location allow' exited with code 255'; Command output: Security exception: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 
[35m[AndroidDriver][39m java.lang.SecurityException: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforce(ContextImpl.java:1831)
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforcePermission(ContextImpl.java:1844)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.setMode(AppOpsService.java:674)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:2026)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService$Shell.onCommand(AppOpsService.java:1800)
[35m[AndroidDriver][39m 	at android.os.ShellCommand.exec(ShellCommand.java:96)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:1974)
[35m[AndroidDriver][39m 	at android.os.Binder.shellCommand(Binder.java:594)
[35m[AndroidDriver][39m 	at android.os.Binder.onTransact(Binder.java:492)
[35m[AndroidDriver][39m 	at com.android.internal.app.IAppOpsService$Stub.onTransact(IAppOpsService.java:341)
[35m[AndroidDriver][39m 	at android.os.Binder.execTransact(Binder.java:697)
[35m[AndroidDriver][39m 
[35m[UiAutomator2][39m Server packages are not going to be (re)installed
[35m[ADB][39m Adding packages ["io.appium.settings","io.appium.uiautomator2.server","io.appium.uiautomator2.server.test"] to Doze whitelist
[35m[UiAutomator2][39m Starting UIAutomator2 server 4.27.0
[35m[UiAutomator2][39m Using UIAutomator2 server from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.27.0.apk' and test from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk'
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m Determined the downstream protocol as 'W3C'
[35m[WD Proxy][39m Got response with status 404: {"sessionId":"e4038960-faa9-47d9-a68a-18dd320a4ddc","value":{"error":"invalid session id","message":"The session identified by e4038960-faa9-47d9-a68a-18dd320a4ddc is not known","stacktrace":"io.appium.uiautomator2.common.exceptions.NoSuchDriverException: The session identified by e4038960-faa9-47d9-a68a-18dd320a4ddc is not known\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:54)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:266)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:260)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:3...
[35m[WD Proxy][39m Got response with status 404: {"sessionId":"e4038960-faa9-47d9-a68a-18dd320a4ddc","value":{"error":"invalid session id","message":"The session identified by e4038960-faa9-47d9-a68a-18dd320a4ddc is not known","stacktrace":"io.appium.uiautomator2.common.exceptions.NoSuchDriverException: The session identified by e4038960-faa9-47d9-a68a-18dd320a4ddc is not known\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:54)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:266)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:260)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:3...
[35m[UiAutomator2][39m Did not get confirmation UiAutomator2 deleteSession worked; Error was: NoSuchDriverError: The session identified by e4038960-faa9-47d9-a68a-18dd320a4ddc is not known
[35m[HTTP][39m [37m<-- POST /wd/hub/session [39m[33m404[39m [90m8400 ms - 3691[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mGET[39m [37m/wd/hub/status[39m
[35m[HTTP][39m [90m{}[39m
[35m[HTTP][39m [37m<-- GET /wd/hub/status [39m[32m200[39m [90m1 ms - 68[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m
[35m[HTTP][39m [90m{"desiredCapabilities":{"appPackage":"com.kfc.mobile","noReset":true,"deviceId":"","deviceName":"OPPO CPH1803 (Android 8.1.0)","platform":"ANDROID","appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","newCommandTimeout":1800,"platformVersion":"8.1.0","automationName":"uiautomator2","autoGrantPermissions":true,"platformName":"Android","udid":"cefdc881","systemPort":8287},"capabilities":{"firstMatch":[{"appium:appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","appium:appPackage":"com.kfc.mobile","appium:autoGrantPermissions":true,"appium:automationName":"uiautomator2","deviceId":"","appium:deviceName":"OPPO CPH1803 (Android 8.1.0)","appium:newCommandTimeout":1800,"appium:noReset":true,"platform":"ANDROID","platformName":"android","appium:platformVersion":"8.1.0","appium:systemPort":8287,"appium:udid":"cefdc881"}]}}[39m
[35m[BaseDriver][39m The following capabilities are not standard capabilities and should have an extension prefix:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m   chromedriverPort
[35m[Appium][39m Appium v1.22.3 creating new AndroidUiautomator2Driver (v1.70.1) session
[35m[BaseDriver][39m The following capabilities were provided, but are not recognized by Appium:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m Session created with session id: 2c89f393-7db9-4046-a117-f493d81a5135
[35m[UiAutomator2][39m Starting 'com.kfc.mobile' directly on the device
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m Retrieving device list
[35m[AndroidDriver][39m Using device: cefdc881
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m No app sent in, not parsing package/activity
[35m[AndroidDriver][39m Unable to set mock location for app 'io.appium.settings': Error executing adbExec. Original error: 'Command ''C:\\Users\\Ahadian Rifky\\.katalon\\tools\\android_sdk\\platform-tools\\adb.exe' -P 5037 -s cefdc881 shell appops set io.appium.settings android\:mock_location allow' exited with code 255'; Command output: Security exception: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 
[35m[AndroidDriver][39m java.lang.SecurityException: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforce(ContextImpl.java:1831)
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforcePermission(ContextImpl.java:1844)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.setMode(AppOpsService.java:674)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:2026)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService$Shell.onCommand(AppOpsService.java:1800)
[35m[AndroidDriver][39m 	at android.os.ShellCommand.exec(ShellCommand.java:96)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:1974)
[35m[AndroidDriver][39m 	at android.os.Binder.shellCommand(Binder.java:594)
[35m[AndroidDriver][39m 	at android.os.Binder.onTransact(Binder.java:492)
[35m[AndroidDriver][39m 	at com.android.internal.app.IAppOpsService$Stub.onTransact(IAppOpsService.java:341)
[35m[AndroidDriver][39m 	at android.os.Binder.execTransact(Binder.java:697)
[35m[AndroidDriver][39m 
[35m[UiAutomator2][39m Server packages are not going to be (re)installed
[35m[ADB][39m Adding packages ["io.appium.settings","io.appium.uiautomator2.server","io.appium.uiautomator2.server.test"] to Doze whitelist
[35m[UiAutomator2][39m Starting UIAutomator2 server 4.27.0
[35m[UiAutomator2][39m Using UIAutomator2 server from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.27.0.apk' and test from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk'
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[UiAutomator2][39m The instrumentation process has been unexpectedly terminated. Retrying UiAutomator2 startup (#1 of 1)
[35m[ADB][39m No 'uiautomator' process has been found
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m Determined the downstream protocol as 'W3C'
[35m[WD Proxy][39m Got response with status 404: {"sessionId":"7fa6ba3d-72fc-47ee-a18e-49d5244ce85d","value":{"error":"invalid session id","message":"The session identified by 7fa6ba3d-72fc-47ee-a18e-49d5244ce85d is not known","stacktrace":"io.appium.uiautomator2.common.exceptions.NoSuchDriverException: The session identified by 7fa6ba3d-72fc-47ee-a18e-49d5244ce85d is not known\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:54)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:266)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:260)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:3...
[35m[WD Proxy][39m Got response with status 404: {"sessionId":"7fa6ba3d-72fc-47ee-a18e-49d5244ce85d","value":{"error":"invalid session id","message":"The session identified by 7fa6ba3d-72fc-47ee-a18e-49d5244ce85d is not known","stacktrace":"io.appium.uiautomator2.common.exceptions.NoSuchDriverException: The session identified by 7fa6ba3d-72fc-47ee-a18e-49d5244ce85d is not known\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:54)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:266)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:260)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:3...
[35m[UiAutomator2][39m Did not get confirmation UiAutomator2 deleteSession worked; Error was: NoSuchDriverError: The session identified by 7fa6ba3d-72fc-47ee-a18e-49d5244ce85d is not known
[35m[HTTP][39m [37m<-- POST /wd/hub/session [39m[33m404[39m [90m11773 ms - 3691[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mGET[39m [37m/wd/hub/status[39m
[35m[HTTP][39m [90m{}[39m
[35m[HTTP][39m [37m<-- GET /wd/hub/status [39m[32m200[39m [90m1 ms - 68[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m
[35m[HTTP][39m [90m{"desiredCapabilities":{"appPackage":"com.kfc.mobile","noReset":true,"deviceId":"","deviceName":"OPPO CPH1803 (Android 8.1.0)","platform":"ANDROID","appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","newCommandTimeout":1800,"platformVersion":"8.1.0","automationName":"uiautomator2","autoGrantPermissions":true,"platformName":"Android","udid":"cefdc881","systemPort":8270},"capabilities":{"firstMatch":[{"appium:appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","appium:appPackage":"com.kfc.mobile","appium:autoGrantPermissions":true,"appium:automationName":"uiautomator2","deviceId":"","appium:deviceName":"OPPO CPH1803 (Android 8.1.0)","appium:newCommandTimeout":1800,"appium:noReset":true,"platform":"ANDROID","platformName":"android","appium:platformVersion":"8.1.0","appium:systemPort":8270,"appium:udid":"cefdc881"}]}}[39m
[35m[BaseDriver][39m The following capabilities are not standard capabilities and should have an extension prefix:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m   chromedriverPort
[35m[Appium][39m Appium v1.22.3 creating new AndroidUiautomator2Driver (v1.70.1) session
[35m[BaseDriver][39m The following capabilities were provided, but are not recognized by Appium:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m Session created with session id: 801fac41-d7da-4145-a39f-6ae423b4efc2
[35m[UiAutomator2][39m Starting 'com.kfc.mobile' directly on the device
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m Retrieving device list
[35m[AndroidDriver][39m Using device: cefdc881
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m No app sent in, not parsing package/activity
[35m[AndroidDriver][39m Unable to set mock location for app 'io.appium.settings': Error executing adbExec. Original error: 'Command ''C:\\Users\\Ahadian Rifky\\.katalon\\tools\\android_sdk\\platform-tools\\adb.exe' -P 5037 -s cefdc881 shell appops set io.appium.settings android\:mock_location allow' exited with code 255'; Command output: Security exception: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 
[35m[AndroidDriver][39m java.lang.SecurityException: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforce(ContextImpl.java:1831)
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforcePermission(ContextImpl.java:1844)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.setMode(AppOpsService.java:674)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:2026)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService$Shell.onCommand(AppOpsService.java:1800)
[35m[AndroidDriver][39m 	at android.os.ShellCommand.exec(ShellCommand.java:96)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:1974)
[35m[AndroidDriver][39m 	at android.os.Binder.shellCommand(Binder.java:594)
[35m[AndroidDriver][39m 	at android.os.Binder.onTransact(Binder.java:492)
[35m[AndroidDriver][39m 	at com.android.internal.app.IAppOpsService$Stub.onTransact(IAppOpsService.java:341)
[35m[AndroidDriver][39m 	at android.os.Binder.execTransact(Binder.java:697)
[35m[AndroidDriver][39m 
[35m[UiAutomator2][39m Server packages are not going to be (re)installed
[35m[ADB][39m Adding packages ["io.appium.settings","io.appium.uiautomator2.server","io.appium.uiautomator2.server.test"] to Doze whitelist
[35m[UiAutomator2][39m Starting UIAutomator2 server 4.27.0
[35m[UiAutomator2][39m Using UIAutomator2 server from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.27.0.apk' and test from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk'
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[UiAutomator2][39m The instrumentation process has been unexpectedly terminated. Retrying UiAutomator2 startup (#1 of 1)
[35m[ADB][39m No 'uiautomator' process has been found
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m Determined the downstream protocol as 'W3C'
[35m[AndroidDriver][39m Screen already unlocked, doing nothing
[35m[UiAutomator2][39m Starting 'com.kfc.mobile/com.kfc.mobile.presentation.splash.SplashActivity and waiting for 'com.kfc.mobile/com.kfc.mobile.presentation.splash.SplashActivity'
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[UiAutomator2][39m Did not get confirmation UiAutomator2 deleteSession worked; Error was: UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
[35m[HTTP][39m [37m<-- POST /wd/hub/session [39m[31m500[39m [90m14377 ms - 1355[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mGET[39m [37m/wd/hub/status[39m
[35m[HTTP][39m [90m{}[39m
[35m[HTTP][39m [37m<-- GET /wd/hub/status [39m[32m200[39m [90m1 ms - 68[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m
[35m[HTTP][39m [90m{"desiredCapabilities":{"appPackage":"com.kfc.mobile","noReset":true,"deviceId":"","deviceName":"OPPO CPH1803 (Android 8.1.0)","platform":"ANDROID","appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","newCommandTimeout":1800,"platformVersion":"8.1.0","automationName":"uiautomator2","autoGrantPermissions":true,"platformName":"Android","udid":"cefdc881","systemPort":8230},"capabilities":{"firstMatch":[{"appium:appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","appium:appPackage":"com.kfc.mobile","appium:autoGrantPermissions":true,"appium:automationName":"uiautomator2","deviceId":"","appium:deviceName":"OPPO CPH1803 (Android 8.1.0)","appium:newCommandTimeout":1800,"appium:noReset":true,"platform":"ANDROID","platformName":"android","appium:platformVersion":"8.1.0","appium:systemPort":8230,"appium:udid":"cefdc881"}]}}[39m
[35m[BaseDriver][39m The following capabilities are not standard capabilities and should have an extension prefix:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m   chromedriverPort
[35m[Appium][39m Appium v1.22.3 creating new AndroidUiautomator2Driver (v1.70.1) session
[35m[BaseDriver][39m The following capabilities were provided, but are not recognized by Appium:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m Session created with session id: 0039950a-f18b-4583-9dde-7c91adc42a01
[35m[UiAutomator2][39m Starting 'com.kfc.mobile' directly on the device
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m Retrieving device list
[35m[AndroidDriver][39m Using device: cefdc881
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m No app sent in, not parsing package/activity
[35m[AndroidDriver][39m Unable to set mock location for app 'io.appium.settings': Error executing adbExec. Original error: 'Command ''C:\\Users\\Ahadian Rifky\\.katalon\\tools\\android_sdk\\platform-tools\\adb.exe' -P 5037 -s cefdc881 shell appops set io.appium.settings android\:mock_location allow' exited with code 255'; Command output: Security exception: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 
[35m[AndroidDriver][39m java.lang.SecurityException: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforce(ContextImpl.java:1831)
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforcePermission(ContextImpl.java:1844)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.setMode(AppOpsService.java:674)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:2026)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService$Shell.onCommand(AppOpsService.java:1800)
[35m[AndroidDriver][39m 	at android.os.ShellCommand.exec(ShellCommand.java:96)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:1974)
[35m[AndroidDriver][39m 	at android.os.Binder.shellCommand(Binder.java:594)
[35m[AndroidDriver][39m 	at android.os.Binder.onTransact(Binder.java:492)
[35m[AndroidDriver][39m 	at com.android.internal.app.IAppOpsService$Stub.onTransact(IAppOpsService.java:341)
[35m[AndroidDriver][39m 	at android.os.Binder.execTransact(Binder.java:697)
[35m[AndroidDriver][39m 
[35m[UiAutomator2][39m Server packages are not going to be (re)installed
[35m[ADB][39m Adding packages ["io.appium.settings","io.appium.uiautomator2.server","io.appium.uiautomator2.server.test"] to Doze whitelist
[35m[UiAutomator2][39m Starting UIAutomator2 server 4.27.0
[35m[UiAutomator2][39m Using UIAutomator2 server from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.27.0.apk' and test from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk'
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m Determined the downstream protocol as 'W3C'
[35m[AndroidDriver][39m Screen already unlocked, doing nothing
[35m[UiAutomator2][39m Starting 'com.kfc.mobile/com.kfc.mobile.presentation.splash.SplashActivity and waiting for 'com.kfc.mobile/com.kfc.mobile.presentation.splash.SplashActivity'
[35m[UiAutomator2][39m Did not get confirmation UiAutomator2 deleteSession worked; Error was: UnknownError: An unknown server-side error occurred while processing the command. Original error: 'DELETE /' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
[35m[HTTP][39m [37m<-- POST /wd/hub/session [39m[31m500[39m [90m11438 ms - 1542[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mGET[39m [37m/wd/hub/status[39m
[35m[HTTP][39m [90m{}[39m
[35m[HTTP][39m [37m<-- GET /wd/hub/status [39m[32m200[39m [90m1 ms - 68[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m
[35m[HTTP][39m [90m{"desiredCapabilities":{"appPackage":"com.kfc.mobile","noReset":true,"deviceId":"","deviceName":"OPPO CPH1803 (Android 8.1.0)","platform":"ANDROID","appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","newCommandTimeout":1800,"platformVersion":"8.1.0","automationName":"uiautomator2","autoGrantPermissions":true,"platformName":"Android","udid":"cefdc881","systemPort":8267},"capabilities":{"firstMatch":[{"appium:appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","appium:appPackage":"com.kfc.mobile","appium:autoGrantPermissions":true,"appium:automationName":"uiautomator2","deviceId":"","appium:deviceName":"OPPO CPH1803 (Android 8.1.0)","appium:newCommandTimeout":1800,"appium:noReset":true,"platform":"ANDROID","platformName":"android","appium:platformVersion":"8.1.0","appium:systemPort":8267,"appium:udid":"cefdc881"}]}}[39m
[35m[BaseDriver][39m The following capabilities are not standard capabilities and should have an extension prefix:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m   chromedriverPort
[35m[Appium][39m Appium v1.22.3 creating new AndroidUiautomator2Driver (v1.70.1) session
[35m[BaseDriver][39m The following capabilities were provided, but are not recognized by Appium:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m Session created with session id: 1f97251b-b099-409a-a348-0a8a5e0608be
[35m[UiAutomator2][39m Starting 'com.kfc.mobile' directly on the device
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m Retrieving device list
[35m[AndroidDriver][39m Using device: cefdc881
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m No app sent in, not parsing package/activity
[35m[AndroidDriver][39m Unable to set mock location for app 'io.appium.settings': Error executing adbExec. Original error: 'Command ''C:\\Users\\Ahadian Rifky\\.katalon\\tools\\android_sdk\\platform-tools\\adb.exe' -P 5037 -s cefdc881 shell appops set io.appium.settings android\:mock_location allow' exited with code 255'; Command output: Security exception: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 
[35m[AndroidDriver][39m java.lang.SecurityException: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforce(ContextImpl.java:1831)
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforcePermission(ContextImpl.java:1844)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.setMode(AppOpsService.java:674)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:2026)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService$Shell.onCommand(AppOpsService.java:1800)
[35m[AndroidDriver][39m 	at android.os.ShellCommand.exec(ShellCommand.java:96)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:1974)
[35m[AndroidDriver][39m 	at android.os.Binder.shellCommand(Binder.java:594)
[35m[AndroidDriver][39m 	at android.os.Binder.onTransact(Binder.java:492)
[35m[AndroidDriver][39m 	at com.android.internal.app.IAppOpsService$Stub.onTransact(IAppOpsService.java:341)
[35m[AndroidDriver][39m 	at android.os.Binder.execTransact(Binder.java:697)
[35m[AndroidDriver][39m 
[35m[UiAutomator2][39m Server packages are not going to be (re)installed
[35m[ADB][39m Adding packages ["io.appium.settings","io.appium.uiautomator2.server","io.appium.uiautomator2.server.test"] to Doze whitelist
[35m[UiAutomator2][39m Starting UIAutomator2 server 4.27.0
[35m[UiAutomator2][39m Using UIAutomator2 server from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.27.0.apk' and test from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk'
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m Determined the downstream protocol as 'W3C'
[35m[WD Proxy][39m Got response with status 404: {"sessionId":"fa2bbf2f-7018-4e36-b650-7bf86887eff6","value":{"error":"invalid session id","message":"The session identified by fa2bbf2f-7018-4e36-b650-7bf86887eff6 is not known","stacktrace":"io.appium.uiautomator2.common.exceptions.NoSuchDriverException: The session identified by fa2bbf2f-7018-4e36-b650-7bf86887eff6 is not known\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:54)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:266)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:260)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:3...
[35m[WD Proxy][39m Got response with status 404: {"sessionId":"fa2bbf2f-7018-4e36-b650-7bf86887eff6","value":{"error":"invalid session id","message":"The session identified by fa2bbf2f-7018-4e36-b650-7bf86887eff6 is not known","stacktrace":"io.appium.uiautomator2.common.exceptions.NoSuchDriverException: The session identified by fa2bbf2f-7018-4e36-b650-7bf86887eff6 is not known\n\tat io.appium.uiautomator2.handler.request.SafeRequestHandler.handle(SafeRequestHandler.java:54)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleRequest(AppiumServlet.java:266)\n\tat io.appium.uiautomator2.server.AppiumServlet.handleHttpRequest(AppiumServlet.java:260)\n\tat io.appium.uiautomator2.http.ServerHandler.channelRead(ServerHandler.java:68)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:366)\n\tat io.netty.channel.AbstractChannelHandlerContext.invokeChannelRead(AbstractChannelHandlerContext.java:352)\n\tat io.netty.channel.AbstractChannelHandlerContext.fireChannelRead(AbstractChannelHandlerContext.java:3...
[35m[UiAutomator2][39m Did not get confirmation UiAutomator2 deleteSession worked; Error was: NoSuchDriverError: The session identified by fa2bbf2f-7018-4e36-b650-7bf86887eff6 is not known
[35m[HTTP][39m [37m<-- POST /wd/hub/session [39m[33m404[39m [90m3292 ms - 3691[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mGET[39m [37m/wd/hub/status[39m
[35m[HTTP][39m [90m{}[39m
[35m[HTTP][39m [37m<-- GET /wd/hub/status [39m[32m200[39m [90m1 ms - 68[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m
[35m[HTTP][39m [90m{"desiredCapabilities":{"appPackage":"com.kfc.mobile","noReset":true,"deviceId":"","deviceName":"OPPO CPH1803 (Android 8.1.0)","platform":"ANDROID","appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","newCommandTimeout":1800,"platformVersion":"8.1.0","automationName":"uiautomator2","autoGrantPermissions":true,"platformName":"Android","udid":"cefdc881","systemPort":8297},"capabilities":{"firstMatch":[{"appium:appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","appium:appPackage":"com.kfc.mobile","appium:autoGrantPermissions":true,"appium:automationName":"uiautomator2","deviceId":"","appium:deviceName":"OPPO CPH1803 (Android 8.1.0)","appium:newCommandTimeout":1800,"appium:noReset":true,"platform":"ANDROID","platformName":"android","appium:platformVersion":"8.1.0","appium:systemPort":8297,"appium:udid":"cefdc881"}]}}[39m
[35m[BaseDriver][39m The following capabilities are not standard capabilities and should have an extension prefix:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m   chromedriverPort
[35m[Appium][39m Appium v1.22.3 creating new AndroidUiautomator2Driver (v1.70.1) session
[35m[BaseDriver][39m The following capabilities were provided, but are not recognized by Appium:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m Session created with session id: 6524a61c-c304-4aaf-867b-9323d190c437
[35m[UiAutomator2][39m Starting 'com.kfc.mobile' directly on the device
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m Retrieving device list
[35m[AndroidDriver][39m Using device: cefdc881
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m No app sent in, not parsing package/activity
[35m[AndroidDriver][39m Unable to set mock location for app 'io.appium.settings': Error executing adbExec. Original error: 'Command ''C:\\Users\\Ahadian Rifky\\.katalon\\tools\\android_sdk\\platform-tools\\adb.exe' -P 5037 -s cefdc881 shell appops set io.appium.settings android\:mock_location allow' exited with code 255'; Command output: Security exception: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 
[35m[AndroidDriver][39m java.lang.SecurityException: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforce(ContextImpl.java:1831)
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforcePermission(ContextImpl.java:1844)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.setMode(AppOpsService.java:674)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:2026)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService$Shell.onCommand(AppOpsService.java:1800)
[35m[AndroidDriver][39m 	at android.os.ShellCommand.exec(ShellCommand.java:96)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:1974)
[35m[AndroidDriver][39m 	at android.os.Binder.shellCommand(Binder.java:594)
[35m[AndroidDriver][39m 	at android.os.Binder.onTransact(Binder.java:492)
[35m[AndroidDriver][39m 	at com.android.internal.app.IAppOpsService$Stub.onTransact(IAppOpsService.java:341)
[35m[AndroidDriver][39m 	at android.os.Binder.execTransact(Binder.java:697)
[35m[AndroidDriver][39m 
[35m[UiAutomator2][39m Server packages are not going to be (re)installed
[35m[ADB][39m Adding packages ["io.appium.settings","io.appium.uiautomator2.server","io.appium.uiautomator2.server.test"] to Doze whitelist
[35m[UiAutomator2][39m Starting UIAutomator2 server 4.27.0
[35m[UiAutomator2][39m Using UIAutomator2 server from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.27.0.apk' and test from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk'
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[UiAutomator2][39m The instrumentation process has been unexpectedly terminated. Retrying UiAutomator2 startup (#1 of 1)
[35m[ADB][39m No 'uiautomator' process has been found
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m Determined the downstream protocol as 'W3C'
[35m[AndroidDriver][39m Screen already unlocked, doing nothing
[35m[UiAutomator2][39m Starting 'com.kfc.mobile/com.kfc.mobile.presentation.splash.SplashActivity and waiting for 'com.kfc.mobile/com.kfc.mobile.presentation.splash.SplashActivity'
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[UiAutomator2][39m Did not get confirmation UiAutomator2 deleteSession worked; Error was: UnknownError: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to the remote server. Original error: socket hang up
[35m[HTTP][39m [37m<-- POST /wd/hub/session [39m[31m500[39m [90m15381 ms - 1355[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mGET[39m [37m/wd/hub/status[39m
[35m[HTTP][39m [90m{}[39m
[35m[HTTP][39m [37m<-- GET /wd/hub/status [39m[32m200[39m [90m1 ms - 68[39m
[35m[HTTP][39m [90m[39m
[35m[HTTP][39m [37m-->[39m [37mPOST[39m [37m/wd/hub/session[39m
[35m[HTTP][39m [90m{"desiredCapabilities":{"appPackage":"com.kfc.mobile","noReset":true,"deviceId":"","deviceName":"OPPO CPH1803 (Android 8.1.0)","platform":"ANDROID","appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","newCommandTimeout":1800,"platformVersion":"8.1.0","automationName":"uiautomator2","autoGrantPermissions":true,"platformName":"Android","udid":"cefdc881","systemPort":8295},"capabilities":{"firstMatch":[{"appium:appActivity":"com.kfc.mobile.presentation.splash.SplashActivity","appium:appPackage":"com.kfc.mobile","appium:autoGrantPermissions":true,"appium:automationName":"uiautomator2","deviceId":"","appium:deviceName":"OPPO CPH1803 (Android 8.1.0)","appium:newCommandTimeout":1800,"appium:noReset":true,"platform":"ANDROID","platformName":"android","appium:platformVersion":"8.1.0","appium:systemPort":8295,"appium:udid":"cefdc881"}]}}[39m
[35m[BaseDriver][39m The following capabilities are not standard capabilities and should have an extension prefix:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m   chromedriverPort
[35m[Appium][39m Appium v1.22.3 creating new AndroidUiautomator2Driver (v1.70.1) session
[35m[BaseDriver][39m The following capabilities were provided, but are not recognized by Appium:
[35m[BaseDriver][39m   deviceId
[35m[BaseDriver][39m   platform
[35m[BaseDriver][39m Session created with session id: d99ca903-029c-40ee-a71c-17c792d4b66e
[35m[UiAutomator2][39m Starting 'com.kfc.mobile' directly on the device
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m Retrieving device list
[35m[AndroidDriver][39m Using device: cefdc881
[35m[ADB][39m Using 'adb.exe' from 'C:\Users\Ahadian Rifky\.katalon\tools\android_sdk\platform-tools\adb.exe'
[35m[AndroidDriver][39m No app sent in, not parsing package/activity
[35m[AndroidDriver][39m Unable to set mock location for app 'io.appium.settings': Error executing adbExec. Original error: 'Command ''C:\\Users\\Ahadian Rifky\\.katalon\\tools\\android_sdk\\platform-tools\\adb.exe' -P 5037 -s cefdc881 shell appops set io.appium.settings android\:mock_location allow' exited with code 255'; Command output: Security exception: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 
[35m[AndroidDriver][39m java.lang.SecurityException: uid 2000 does not have android.permission.UPDATE_APP_OPS_STATS.
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforce(ContextImpl.java:1831)
[35m[AndroidDriver][39m 	at android.app.ContextImpl.enforcePermission(ContextImpl.java:1844)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.setMode(AppOpsService.java:674)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:2026)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService$Shell.onCommand(AppOpsService.java:1800)
[35m[AndroidDriver][39m 	at android.os.ShellCommand.exec(ShellCommand.java:96)
[35m[AndroidDriver][39m 	at com.android.server.AppOpsService.onShellCommand(AppOpsService.java:1974)
[35m[AndroidDriver][39m 	at android.os.Binder.shellCommand(Binder.java:594)
[35m[AndroidDriver][39m 	at android.os.Binder.onTransact(Binder.java:492)
[35m[AndroidDriver][39m 	at com.android.internal.app.IAppOpsService$Stub.onTransact(IAppOpsService.java:341)
[35m[AndroidDriver][39m 	at android.os.Binder.execTransact(Binder.java:697)
[35m[AndroidDriver][39m 
[35m[UiAutomator2][39m Server packages are not going to be (re)installed
[35m[ADB][39m Adding packages ["io.appium.settings","io.appium.uiautomator2.server","io.appium.uiautomator2.server.test"] to Doze whitelist
[35m[UiAutomator2][39m Starting UIAutomator2 server 4.27.0
[35m[UiAutomator2][39m Using UIAutomator2 server from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-v4.27.0.apk' and test from 'C:\Users\Ahadian Rifky\AppData\Roaming\npm\node_modules\appium\node_modules\appium-uiautomator2-server\apks\appium-uiautomator2-server-debug-androidTest.apk'
[35m[UiAutomator2][39m Waiting up to 30000ms for UiAutomator2 to be online...
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m socket hang up
[35m[HTTP][39m [37m<-- POST /wd/hub/session [39m[32m-[39m [90m- ms - -[39m
[35m[HTTP][39m [90m[39m
[35m[WD Proxy][39m socket hang up
[35m[WD Proxy][39m Determined the downstream protocol as 'W3C'
[35m[AndroidDriver][39m Screen already unlocked, doing nothing
[35m[UiAutomator2][39m Starting 'com.kfc.mobile/com.kfc.mobile.presentation.splash.SplashActivity and waiting for 'com.kfc.mobile/com.kfc.mobile.presentation.splash.SplashActivity'
[35m[UiAutomator2][39m Did not get confirmation UiAutomator2 deleteSession worked; Error was: UnknownError: An unknown server-side error occurred while processing the command. Original error: 'DELETE /' cannot be proxied to UiAutomator2 server because the instrumentation process is not running (probably crashed). Check the server log and/or the logcat output for more details
